Tabloid/client/src/components/TagCreate.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState  } from 'react';
+import { addTag } from '../modules/TagManagment';
+import { Link } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+
+export const CreateTag=()=>{
+    const history = useHistory();
+    const TagForm={
```

Comment:
Is there anything in this component that prevents the user from submitting an empty tag name?

Reply:
@joshdbarton  Added an if statement inside the handelSaveFunction that looks for an empty string and trim() to stop a user from entering in spaces